package-lock.json

```diff
@@ -0,0 +1,2127 @@
+{
```

Comment:
Not sure if we want the `package-lock.json` committed or if that's going to cause issues when its used within other projects running on various versions of node.